plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1010,6 +1040,32 @@ def get_want_authentication_policy_server(self, auth_policy_server):
                         "ipAddress": ip_address
                     })
 
+                    fqdn = ise_credential.get("fqdn")
+                    if not fqdn:
+                        if not auth_server_exists:
+                            self.msg = "The required parameter 'fqdn' is missing when 'server_type' is 'ISE'."
+                            self.status = "failed"
+                            return self
+
```

Comment:
self.log(
        "Auth server exists; retrieving authentication and policy servers to obtain FQDN for IP '{0}'..."
        .format(ip_address), 
        "DEBUG"
    )